src/Command/Team/TeamListCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+namespace Platformsh\Cli\Command\Team;
+
+use Platformsh\Cli\Console\ProgressMessage;
+use Platformsh\Cli\Service\Table;
+use Platformsh\Client\Model\Team\Team;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class TeamListCommand extends TeamCommandBase
+{
+    private $tableHeader = [
+        'id' => 'ID',
+        'label' => 'Label',
+        'member_count' => '# Users',
+        'project_count' => '# Projects',
+        'project_permissions' => 'Permissions',
+        'created_at' => 'Created at',
+        'updated_at' => 'Updated at',
+    ];
+    private $defaultColumns = ['id', 'label', 'member_count', 'project_count', 'project_permissions'];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('team:list')
+            ->setAliases(['teams'])
+            ->setDescription('List teams')
+            ->addOption('count', null, InputOption::VALUE_REQUIRED, 'The number of items to display per page. Use 0 to disable pagination.')
+            ->addOption('sort', null, InputOption::VALUE_REQUIRED, 'A team property to sort by', 'label')
+            ->addOption('reverse', null, InputOption::VALUE_NONE, 'Sort in reverse order')
+            ->addOrganizationOptions(true);
+        Table::configureInput($this->getDefinition(), $this->tableHeader, $this->defaultColumns);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $organization = $this->validateOrganizationInput($input);
```

Comment:
It'd be useful to catch the `NoOrganizationsFoundException` and recommend creating an org